test/python/WMCore_t/WorkQueue_t/Policy_t/Start_t/Dataset_t.py

```diff
@@ -85,116 +82,110 @@ def testWhiteBlackLists(self):
         Tier1ReRecoWorkload = factory.factoryWorkloadConstruction('ReRecoWorkload', rerecoArgs)
         Tier1ReRecoWorkload.setStartPolicy('Dataset', **self.splitArgs)
         inputDataset = getFirstTask(Tier1ReRecoWorkload).inputDataset()
-        dataset = "/%s/%s/%s" % (inputDataset.primary,
-                                     inputDataset.processed,
-                                     inputDataset.tier)
-        dbs = {inputDataset.dbsurl : DBSReader(inputDataset.dbsurl)}
```

Comment:
I don't think you can remove this line as I said in chat. while we never use "dbs", there is probably an effect to calling DBSReader

Just change dbs to dummyDBS to get the complaint to stop.